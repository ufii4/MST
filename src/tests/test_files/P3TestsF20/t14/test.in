n 5
i 3;4;4
d 4;3
edge_count
i 1;2;1.1
i 2;3;5
i 4;3;1.3
i 4;2;1.3
i 1;0;2.1
mst
i 2;4;0.8
mst
d 4;2
mst
clear
i 2;1;0.7
i 1;0;0.3
i 0;2;1.1
i 3;4;2.2
mst
i 4;1;0.9
mst
exit
