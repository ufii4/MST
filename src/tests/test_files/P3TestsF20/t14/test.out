success
success
success
edge count is 0
success
success
success
success
success
mst 5.80
success
mst 5.30
success
mst 9.50
success
success
success
success
success
not connected
success
mst 4.10
